content/2023-12-28-embedded-hal-v1.md

```diff
@@ -0,0 +1,77 @@
++++
+title = "embedded-hal v1.0 now released!"
+date = 2023-12-28
+draft = false
+in_search_index = true
+template = "page.html"
++++
+
+The Rust Embedded Working Group is proud to announce the release of `embedded-hal` version 1.0 together with the
+companion crates `embedded-hal-bus`, `embedded-hal-async` and `embedded-hal-nb`.
+
+Check out the [repository](https://github.com/rust-embedded/embedded-hal), the [API documentation](https://github.com/rust-embedded/embedded-hal#crates) and the [migration guide](https://github.com/rust-embedded/embedded-hal/blob/master/docs/migrating-from-0.2-to-1.0.md).
+
+The `embedded-hal` crates provide traits (interfaces) for using peripherals commonly available in microcontrollers
+such as GPIO, UART, SPI or I2C. They allow writing drivers (for sensors, displays, actuators, network adapters, etc.) in
+a generic way, so they work on any microcontroller with an `embedded-hal` implementation without
+modifying them. It's a central piece of the Embedded Rust ecosystem, ensuring interoperability throughout.
+
+The 1.0 release has been in the works since 2020. Now that it's out, we consider all
+traits in it to be *stable*. The plan is to extend `embedded-hal` with more traits in future 1.x releases,
+not doing more breaking changes (i.e. there are no plans for a a 2.0 release). This will provide a stable
+base for building HALs and drivers.
+
+So, what's new in `embedded-hal` 1.0?
+
+<!-- more -->
+
+## Focus on drivers
+
+Previous versions of `embedded-hal` had a dual goal of standardizing HAL APIs for end users, and allowing writing generic drivers. 
+Experience has shown that these goals sometimes conflict with each other. As the latter brings much more value, 1.0 focuses on that.
+
+We've simplified some traits and merged others to remove interoperability gotchas.
+
+We've removed traits that were found to not be usable for generic drivers (most notably timers). The plan is to add
+them back in the future, with a better design. See the [migration guide](https://github.com/rust-embedded/embedded-hal/blob/master/docs/migrating-from-0.2-to-1.0.md) for details and links to the tracking issues.
+
+## Async
+
+A new addition is the `embedded-hal-async` crate, containing async versions of the traits. With today's Rust 1.75 release,
```

Comment:
maybe it'd make sense to add a link here to the rust 1.75 announcement (once it's out and the URL is known)? might help some users interested in it to find further details on the newly stabilised async features mentioned here